### DISCUSSION TITLE
How to use an external library in a zephyr app

### DISCUSSION THREAD

**Discussion by zzzh** [zephyrproject-rtos/zephyr#37593] [Q&A]
Seems target_link_libraries(...) don't work...

**Comment by yashi** [comment]
Please explain a bit more.  Help us to help you.
http://www.catb.org/esr/faqs/smart-questions.html

**Comment by achandr3** [comment]
WE NEED HELP HOW TO LINK EXTERNAL STATIC LIBARIRES TO ZEPHYR APP

**Reply by gmarull** [reply]
What are “LIBARIRES”?

**Comment by henrikbrixandersen** [comment]
Please see https://docs.zephyrproject.org/latest/samples/application_development/external_lib/README.html (and https://github.com/zephyrproject-rtos/zephyr/tree/main/samples/application_development/external_lib).

Also, please do not write in ALL-CAPS.

**Reply by achandr3** [reply]
Thank you. I have static library being generated from segger , which i want to link within zephyr frame work, will the above we can consider for building?

**Reply by henrikbrixandersen** [reply]
Have you tried it?
